github/GithubObject.py

```diff
@@ -151,49 +152,51 @@ def __init__(
         if self.CHECK_AFTER_INIT_FLAG:  # pragma no branch (Flag always set in tests)
             requester.check_me(self)
 
-    def _storeAndUseAttributes(self, headers: Dict[str, Union[str, int]], attributes: Any) -> None:
+    def _storeAndUseAttributes(self, headers: dict[str, str | int], attributes: Any) -> None:
         # Make sure headers are assigned before calling _useAttributes
         # (Some derived classes will use headers in _useAttributes)
         self._headers = headers
         self._rawData = attributes
         self._useAttributes(attributes)
 
     @property
-    def raw_data(self) -> Dict[str, Any]:
+    def raw_data(self) -> dict[str, Any]:
         """
         :type: dict
         """
-        self._completeIfNeeded()
+        if isinstance(self, CompletableGithubObject):
+            self._completeIfNeeded()
```

Comment:
Why not do this through override, instead of instance check?

Reply:
As in 4bd755fefbc6125fb721724a94f2f8285db1e960?